CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## [1.2.1] - 2022-03-16
+
+### Changed
+- The SPIRE Agent `fetch jwt` CLI command now supports JSON output (#2650)
+- OIDC Discovery Provider now includes the `alg` parameter in JWKs to increase compatibility  (#2771)
+- SPIRE server logs additional authorization information with RPC requests (#2776)
+- Small documentation improvements (#2746, #2792)
+
+### Fixed
+- SPIRE Server properly rotates signing keys when prepared or activated keys are lost from the database (#2770)
```

Comment:
Since this is read as "Fixed ...." I think we should say here something like "SPIRE Server _now_ properly rotates" or "SPIRE Server to properly rotate"

```suggestion
- SPIRE Server now properly rotates signing keys when prepared or activated keys are lost from the database (#2770)
```